subsys/mgmt/mcumgr/transport/Kconfig.bluetooth

```diff
@@ -24,13 +24,24 @@ config MCUMGR_TRANSPORT_BT_REASSEMBLY
 	  When enabled, the SMP BT transport will buffer and reassemble received
 	  packet fragments before passing it for further processing.
 
-config MCUMGR_TRANSPORT_BT_AUTHEN
-	bool "Authenticated requirement for Bluetooth mcumgr SMP transport"
-	depends on BT_SMP
-	default y
+choice MCUMGR_TRANSPORT_BT_PERM
+	prompt "Permission used for the SMP service"
+	default MCUMGR_TRANSPORT_BT_PERM_RW
```

Comment:
This changes the existing behaviour, it should stay with how the previous was if BT_SMP is enabled and call back to this if if it is not

Reply:
All projects that use this module disable the `CONFIG_MCUMGR_TRANSPORT_BT_AUTHEN` so I was suggesting to change the default here. However, the intention of this PR is not to change the defaults (it can be done in a follow-up PR) but to add the possibility of requiring encryption without authentication permissions for the SMP service.

I will restore the default configuration.

Reply:
addressed